### DISCUSSION TITLE
Daily 18/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#173] [General]
Ayer estuve en la clase de full-stack para ver qué cosas nuevas se pedían pero no creo que valga la pena modificar lo que ya entregamos (excepto por lo del script de sql que agregamos). También escribí algunos user stories para llegar a 20 que era otra de las cosas que pedían. Piden codificación de las clases de Python pero no piden que se una al front-end así que se podría armar las clases con código que no haga nada aunque es difícil demostrar que lo que uno hizo está bien, se podría armar el back-end para línea de comando o se podría usar FastAPI, voy a ir leyendo sobre eso para ver qué tan difícil sería hacerlo de esa manera.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve haciendo el script de base de datos, poniendome al día con Inglés y avancé un poco con el estudio de GP. También participé en la clase de Full, por la noche.
Hoy cuento con poco tiempo así que estaré mas que nada estudiando para la evaluación  de Gestión  de Proyecto.

**Comment by claubott** [comment]
Ayer estuve estudiando el material de gestión de proyectos. Hoy estoy con JS.

**Comment by Brian-Almada** [comment]
Ayer no pude tocar nada del estudio. Recién ahora me siento después de un par de días de no estudiar, ni practicar, ni ver nada del proyecto. Ahora me voy a poner un poco con ingles y el resto del tiempo con algo de Python. Espero remontar lo que queda de semana.

**Comment by IngridAlcocer05** [comment]
Ayer me uní a la clase de full-stack. Hoy repase los contenidos frente al próximo parcial de gestión de proyectos.
